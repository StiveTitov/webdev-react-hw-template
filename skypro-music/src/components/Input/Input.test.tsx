import renderer from "react-test-renderer";
import { Input } from ".";
import styles from './Search.module.css'

describe('Компанент- полосы регулировки звука', () => {
    it('корректно отрисовывает разметку с начальными значениями', () => {
      // Тестовый код
      const mockonChange= jest.fn();
      const mocksearchValue= jest.fn();
      const component = renderer.create(<Input 
        className={styles.search__text}
        type={"search"}
        placeholder={"Поиск"}
        name={"search"}
        value={mocksearchValue}
        min={50}
        max={90}
        onChange={mockonChange} />).toJSON();
  expect(component).toMatchSnapshot();
    });
   
    
   });