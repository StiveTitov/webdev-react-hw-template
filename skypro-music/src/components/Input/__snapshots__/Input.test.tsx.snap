// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компанент- полосы регулировки звука корректно отрисовывает разметку с начальными значениями 1`] = `
<input
  className="search__text"
  max={90}
  min={50}
  name="search"
  onChange={[MockFunction]}
  placeholder="Поиск"
  type="search"
  value={[MockFunction]}
/>
`;
